mlfinlab/cross_validation/cross_validation.py

```diff
@@ -1,11 +1,11 @@
 """
-Implements the book chapter 7 on Cross Validation for financial data
+Implements the book chapter 7 on Cross Validation for financial data.
```

Comment:
In the future, please create a new branch for features that aren't directly related to your main branches purposes. 

In this case the main branch is feature importance focusing on Ch8. This piece of code relates more to Ch7 and a previous branch that is no longer active. I would create a new branch, add this changes then make a PR. This will help us to keep the PRs more isolated and easier to revert to if changes need to be "un-maid". 

For now this is fine. Lets keep it in mind going forward.